CMakeLists.txt

```diff
@@ -1,20 +1,14 @@
 cmake_minimum_required(VERSION 3.1...3.17 FATAL_ERROR)
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/Cmake/")
```

Comment:
Your problem is here. It works on windows because filesystems are case insensitive, but on Linux they are. Do on Linux this line is meaningless since `Cmake` is not a directory, but `CMake` is. Either way this is a typo.

Reply:
Thank you! I'll update this right away!